W/wolfSSL/build_tarballs.jl

```diff
@@ -6,32 +6,39 @@ const YGGDRASIL_DIR = "../.."
 include(joinpath(YGGDRASIL_DIR, "platforms", "macos_sdks.jl"))
 
 name = "wolfSSL"
-version = v"5.7.2"
+version = v"5.8.4"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/wolfSSL/wolfssl.git", "00e42151ca061463ba6a95adb2290f678cbca472"),
-    DirectorySource("./bundled"),
+    GitSource("https://github.com/wolfSSL/wolfssl.git", "59f4fa568615396fbf381b073b220d1e8d61e4c2"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/wolfssl*
-atomic_patch -p1 ../patches/mingw32-link-with-ws2_32.patch
 
-mkdir build && cd build
-
-cmake .. \
--DCMAKE_INSTALL_PREFIX=${prefix} \
--DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
--DCMAKE_BUILD_TYPE=Release \
--DWOLFSSL_EXAMPLES=no \
--DWOLFSSL_CRYPT_TESTS=no \
--DBUILD_SHARED_LIBS=ON
```

Comment:
Can you elaborate on why you moved away from CMake?

Reply:
(and Intel ASM feature is also not available)